drive/samples/copy_file.bal

```diff
@@ -43,14 +43,14 @@ public function main() {
             refreshToken: refreshToken
         }
     };
-    drive:Client driveClient = new (config);
+    drive:Client driveClient = checkpanic new (config);
```

Comment:
what is the reason for `checkpanic`?

Reply:
Hi @indikasampath2000 , Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/81 
/Thanks